grpc-service/Dockerfile

```diff
@@ -2,5 +2,5 @@ FROM openjdk:8
 
 ADD target/grpc-service-1.0-SNAPSHOT.jar /home/app.jar
 ADD rockyou.tar.gz /home
-
+CMD tar xf /home/rockyou.tar.gz /home
```

Comment:
Yes, I'm using macos. actually it extracts automatically in case of .tar file (not .tar.gz). 

I'll change it to RUN and change xf to -xvf as per your request.